CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v1.43.0
+------------------------------
+*April 12, 2019*
+- Added `default--scaleUp` size to ratings component.
```

Comment:
Mega nit. Needs an `### Added` title and a line break between the date and the additions.